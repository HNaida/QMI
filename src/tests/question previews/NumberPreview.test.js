import {configure, shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react'
import NumberPreview from '../../components/previews/NumberPreview'

configure({adapter: new Adapter()});

describe('<NumberPreview/>', () => {
    it('renders a number question', () => {
        const question = {
            id: "v9",
            type: "number",
            title: "untitled number",
            placeholder: 5132,
            min: 10,
            max: 100,
            maxlength: 4,
        };
        const component = shallow(<NumberPreview question={question}/>);
        expect(component).toMatchSnapshot();
    });
});
