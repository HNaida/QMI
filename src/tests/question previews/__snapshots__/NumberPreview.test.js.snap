// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NumberTypePreview/> renders a number question 1`] = `ShallowWrapper {}`;
